notebooks/03_embedding_graph_structured_data_h2.ipynb

```diff
@@ -45,13 +45,22 @@
   },
```

Comment:
<p>Just realized that the xticks and yticks from -1 to 1 do not really make sense: this disk is not an isometric representation of the Hyperbolic plane and the ticks are confusing. I know you did not implement this visualization, but would it possible to remove them still?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>